### ISSUE TITLE
Min gold not always 200

### ISSUE THREAD

**Issue by angband-import** [angband/angband#994]
**Reported by CunningGabe on 6 Feb 2008 21:01 UTC**
The recent change to birth.c leaves the following:

```
436	/* Minimum 200 gold */ 
437     if (gold < 100) gold = 200;
```

The only instance of 100 should be 200 as well.

**Comment by angband-import** [comment]
**Comment by takkaria on 8 Feb 2008 00:58 UTC**
Sometimes my own incompetence stuns me. :)

**Comment by angband-import** [comment]
**Comment by takkaria on 8 Feb 2008 13:28 UTC**
fixed in [1dd8c6d] (SVN r700), thanks for the report.
